src/common/constants/api.js

```diff
@@ -1,11 +1,12 @@
 export const ACQUISITIONS_UNITS_API = 'acquisitions-units/units';
+export const CONFIG_API = 'configurations/entries';
 export const INVOICE_API = 'invoice/invoices';
 export const INVOICE_DOCUMENTS_API = '/documents';
 export const INVOICE_LINE_API = 'invoice/invoice-lines';
-export const VENDORS_API = 'organizations-storage/organizations';
-export const CONFIG_API = 'configurations/entries';
+export const PO_LINES_API = 'orders-storage/po-lines';
```

Comment:
Is there a reason to call `orders-storage` API ?

Reply:
Switched on `mod-orders`